### DISCUSSION TITLE
Is there any way to edit files inside hass.io add-on?

### DISCUSSION THREAD

**Discussion by jeugen1** [Koenkk/zigbee2mqtt#6957] [General]
I'm using a zigbee2mqtt as hass.io add-on on raspberry pi. 
Is there any way to get access to devices.js in a container and edit it?

**Comment by McGiverGim** [comment]
I think you don't need to do that. There is an option to add "external converters".

Look here: https://github.com/zigbee2mqtt/hassio-zigbee2mqtt/blob/master/zigbee2mqtt/DOCS.md 
and here: https://www.zigbee2mqtt.io/information/configuration.html#external-converters-configuration

But if you want to access the files inside the addon, you can disable the protection mode in the ssh addon and execute a `docker exec -ti name_of_the_addon bash`

**Reply by jeugen1** [reply]
Thanks for your answer!

I'v installed ssh add-on but using ssh I can connect only to home assistant container, I can't connect to host and execute docker. May be I did not disabled protection mode?

**Reply by McGiverGim** [reply]
Easy to see: type `docker ps` and it will show you a list of all the docker availables. If this command fails then you have protection mode enabled.

**Reply by jeugen1** [reply]
In fact I do not have docker at all...
```
~ $ docker
-bash: docker: command not found
```
I've installed HA using official image, not "supervised". And I still can't find the way to get access to docker. :(
